crates/core/src/node/in_memory.rs

```diff
@@ -450,7 +450,7 @@ impl InMemoryNode {
             .unwrap_or_default();
 
         let verbosity = get_shell().verbosity;
-        if !call_traces.is_empty() && verbosity >= 2 {
+        if !call_traces.is_empty() && verbosity >= 2 && !inner.config.offline {
```

Comment:
This would disable call traces for offline mode fully, right? I guess what I suggested above will not have this drawback as well.